cli/utils.py

```diff
@@ -0,0 +1,11 @@
+"""
+Provide utils for command line interface.
+"""
+import json
+
+
+def dict_to_pretty_json(data):
+    """
+    Convert dictionary to string with indents as human readable text.
+    """
+    return json.dumps(data, indent=4, sort_keys=True)
```

Comment:
Instead of creating a file that already in the develop branch. Just update the branch.

<img width="733" alt="Screen Shot 2019-04-18 at 6 43 32 PM" src="https://user-images.githubusercontent.com/22666467/56373568-e7852080-6209-11e9-90cf-34d6da637dbd.png">

Reply:
Done.